task_03.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""This is week8 synthesizing task_3"""
+
+import decimal
+
+NAME = raw_input('What is your Name? ')
+PRINCIPAL = int(raw_input('What is the Ammount of your Principal? '))
+YEARS = int(raw_input('For how many years is Loan borrowed? '))
+PREQ = raw_input('Are you Pre-qualified? ').lower()
```

Comment:
I think you forgot to convert this value to a boolean. You set it to lower but `no|n|yes|y` are all still strings that are non-empty meaning `PREQ` was never `False`